app/models/miq_report/formatting.rb

```diff
@@ -191,6 +191,23 @@ def format_datetime(val, options)
     val.strftime(options[:format])
   end
 
+  def format_relative_date(val, options)
+    return val unless val.kind_of?(Time) || val.kind_of?(Date)
+    seconds_ago = Time.now - val
+    case seconds_ago
+    when 0...60
+      return "#{seconds_ago.to_i} seconds ago"
+    when 60...3600
+      return "#{(seconds_ago/60).to_i} minutes ago"
+    when 3600...86400
+      return "#{(seconds_ago/3600).to_i} hours ago"
+    when 86400...31536000
+      return "#{(seconds_ago/86400).to_i} days ago"
+    else
+      return "#{(seconds_ago/31536000).to_i} years ago"
+    end
```

Comment:
@mahantidinesh7 regarding https://github.com/ManageIQ/manageiq/pull/22426#issuecomment-1483108961 I think this entire block can be replaced with the Rails helper, so we don't need to have custom logic...something like:

```
helper = Module.new { |m| m.extend(ActionView::Helpers::DateHelper) }
"#{helper.time_ago_in_words(val)} ago"
```

Reply:
Already pinged Dinesh on this one, we can use `ApplicationController.helper` or the one you referenced.

I'm sad that `NumberHelper` is in `ActiveSupport` but `DateHelper` is not. Seems those 2 `_in_words` methods are very common.

I was trying to drop `ApplicationController.helper` but maybe that will need to live on.

Reply:
> I'm sad that NumberHelper is in ActiveSupport but DateHelper is not. Seems those 2 _in_words methods are very common.

Yeah, I was suprised to see NumberHelper made it's way to ActiveSupport - wonder if we can convince them to do the same with this DateHelper also.

Reply:
@Fryguy implemented the request changes.Thanks